### PR TITLE
feat :  done with home page for quiz

### PR DESCRIPTION
**Pull request by iprankurpandey**

### CODE REVIEW

File: homepage.css [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    flex-direction: column;
+    padding: 1rem;
+    margin: 1rem;
+    color: tomato;
+    width: 30rem;
+}
+
+.head-text {
+    font-size: 1rem;
+    color: rgb(248, 63, 63);
```
Comment (Shivani-1524) [inline homepage.css:36]: use color variables in place of rgb values

File: homepage.css [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    font-weight: bold;
+    margin: 0.5rem 0 0.5rem 0;
```
Comment (Shivani-1524) [inline homepage.css:38]: margin: 0.5rem 0 would have the same effect as this

File: homepage.css [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+.mtext {
+    font-size: 3rem;
+    color: rgb(99, 98, 96);
+    font-weight: 600;
+}
+
+.logo-text {
+    font-size: 2rem;
+    color: rgb(247, 2, 96);
```
Comment (Shivani-1524) [inline homepage.css:49]: use color variables everywhere

File: homepage.html [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HomePage</title>
+    <link rel="stylesheet" href="/homepage.css">
+</head>
+
+<body>
+    <div class="navigation-menu">
```
Comment (Shivani-1524) [inline homepage.html:13]: use semantics here such as <nav></nav>

File: homepage.html [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+                </span></div> -->
+                <button class="btn btn-primary-outline">Login</button>
+            <div class="user-account" title="your account details"><span class="material-icons mtrlshop navmtrl">
+                    account_circle
+                </span>
+            </div>
+        </div>
+    </div>
+
+    <div class="main-container">
```
Comment (Shivani-1524) [inline homepage.html:49]: use <main></main> here for better semantics

**Review by Shivani-1524 — CHANGES_REQUESTED** [review]

File: homepage.css [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+}
+
+.text-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    padding: 1rem;
+    margin: 1rem;
+    color: tomato;
```
Comment (yaswanthmyneni) [inline homepage.css:30]: changes: use css variables for all colors.

File: homepage.css (hunk 2 above)
Comment (yaswanthmyneni) [inline homepage.css:38]: suggestion: you can use px units for less than 1rem.

File: homepage.css [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+    border-radius: 10px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    margin: 1rem;
+    box-shadow: 0px 0px 10px rgb(0, 0, 0, 0.5);
+    font-weight: bold;
+    cursor: pointer;
+    /* font-style: italic; */
```
Comment (yaswanthmyneni) [inline homepage.css:129]: changes: remove unused css.

File: homepage.css [hunk 8]
```diff
@@ -0,0 +1,222 @@
+@import url('https://agri-component.netlify.app/css/components.css');
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+.main-container {
+    display: flex;
+    flex-direction: row;
```
Comment (pradipgharge) [inline homepage.css:10]: default flex-direction is row, need not specify it separately.

File: homepage.css (hunk 7 above)
Comment (pradipgharge) [inline homepage.css:129]: Remove unnecessary comments

**Review by yaswanthmyneni — APPROVED** [review]
UI looks good, you can merge after making changes.

File: homepage.css [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+
+.brand-about {
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    flex-direction: column;
+}
+
+
+
```
Comment (pradipgharge) [inline homepage.css:178]: unusual spacing here

File: homepage.css [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+/* footer */
+
+.footer-items {
+    display: flex;
+    /* justify-content: center; */
```
Comment (pradipgharge) [inline homepage.css:183]: remove commented code before pushing

File: homepage.html [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+        <div class="third-brand brand-logo">
+                
+                <img src="https://i.ibb.co/ZKZrDsy/icons8-javascript-480.png" alt="css" class="img-conatiner">
+                <div class="brand-about"> <div class="head-text"> JS Quiz</div> 
+                    <p> JavaScript is a high-level, interpreted programming language. It is a language which is  dynamic. It is also an object-oriented language.
+                    </p>
+                    <button class="play-btn-quiz">PLAY</button>
+                </div>
+
+
```
Comment (pradipgharge) [inline homepage.html:114]: remove extra spacing here

**Review by pradipgharge — CHANGES_REQUESTED** [review]
Suggested some changes, do check

**Pull request merged**